applications/models.py

```diff
@@ -218,6 +218,10 @@ def get_by_rsvp_code(self, code, page):
     def is_accepted(self):
         return self.state == 'accepted'
 
+    @property
+    def city(self):
```

Comment:
Adding properties that are using values from related tables should be done with special care. Calling this property will create additional database calls and we probably don't want that.